modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/configuration/KBServiceConfiguration.java

```diff
@@ -35,4 +35,10 @@ public interface KBServiceConfiguration {
 	@Meta.AD(deflt = "15", name = "check-interval", required = false)
 	public int checkInterval();
 
+	@Meta.AD(
+		deflt = "1", description = "expiration-date-notification-help",
+		name = "expiration-date-notification", required = false
+	)
+	public int expirationDateNotification();
+
```

Comment:
The method name is a bit unclear about the value it returns. I think we should add something more to it (e.g. `expirationDateNotificationWeekInterval()`.